src/react/components/pages/predict/predictResult.tsx

```diff
@@ -164,6 +184,48 @@ export default class PredictResult extends React.Component<IPredictResultProps,
         document.body.appendChild(fileLink);
         fileLink.click();
     }
+    private getItems() {
+        const {tags, predictions} = this.props;
+        const tagsDisplayOrder = tags.map((tag) => tag.name);
+        for (const name of Object.keys(predictions)) {
+            const prediction = predictions[name];
+            if (prediction != null) {
+                prediction.fieldName = name;
+                prediction.displayOrder = tagsDisplayOrder.indexOf(name);
+            }
+        }
+        // not sure if we decide to filter item by the page
+        const items = Object.values(predictions).filter(Boolean).sort((p1, p2) => p1.displayOrder - p2.displayOrder);
+        return items;
+    }
+    private triggerCSVDownload = (): void => {
+        const items = this.getItems();
+        const cols:string=",,,,,,"
+        let csvContent: string = `key,value${cols}`;
+        items.forEach(item => {
+            csvContent += `"${item.fieldName}",`;
```

Comment:
question: it seems only fieldName is added to one row for each item, is this expected?